### PR TITLE
DOCS-1441: Update I2Cs for AMS encoder

### PR DESCRIPTION
**Pull request by sguequierre**
* Remove board 
* Take new screenshot 
* Make index

### CODE REVIEW

**Review by penguinland — APPROVED** [review]
Everything else LGTM! Sorry to make you redo the screenshot, which sounds annoying. :confounded:

File: assets/components/encoder/configure-ams.png
[the remark this answers is not in the document]
Comment (sguequierre) [reply assets/components/encoder/configure-ams.png:1]: No worries!

File: docs/components/encoder/ams-as5048.md [hunk 1]
```diff
@@ -59,8 +56,7 @@ The following attributes are available for `AMS-AS5048` encoders:
 <!-- prettier-ignore -->
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
-| `board` | string | **Required** | The `name` of the [board](/components/board/) to which your encoder is connected. |
 | `connection_type` | string | **Required** | Use `"i2c"`. |
-| `i2c_attributes` | object | **Required** | The attributes to configure [I<sup>2</sup>C](/components/board/#i2cs) connection: <ul> <li> <code>i2c_bus</code>: The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this encoder. <br> Example: `"main"` </li> <li> <code>i2c_addr</code>: The address of the bus. <br> Example: `64` </li> </ul> |
+| `i2c_attributes` | object | **Required** | The attributes to configure [I<sup>2</sup>C](/components/board/#i2cs) connection: <ul> <li> <code>i2c_bus</code>: The index of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this encoder. <br> Example: `"1"` </li> <li> <code>i2c_addr</code>: The address of the bus. <br> Example: `64` </li> </ul> |
```
Comment (andf-viam) [inline docs/components/encoder/ams-as5048.md:60]:
```suggestion
| `i2c_attributes` | object | **Required** | The attributes to configure the [I<sup>2</sup>C](/components/board/#i2cs) connection: <ul> <li> <code>i2c_bus</code>: The index of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this encoder. <br> Example: `"1"` </li> <li> <code>i2c_addr</code>: The address of the bus. <br> Example: `64` </li> </ul> |
```

**Review by andf-viam — APPROVED** [review]
LGTM, thanks!

**Pull request merged**